runners/core-java/src/test/java/org/apache/beam/runners/core/SimpleDoFnRunnerTest.java

```diff
@@ -68,7 +68,8 @@
 @RunWith(JUnit4.class)
 @SuppressWarnings({
   "rawtypes", // TODO(https://github.com/apache/beam/issues/20447)
-  "unused" // TODO(BEAM-13271): Remove when new version of errorprone is released (2.11.0)
+  "unused" // TODO(https://github.com/apache/beam/issues/21230): Remove when new version of
+  // errorprone is released (2.11.0)
```

Comment:
Same here, I wonder if there's an easy way to clean these up (not a big deal if not).

Reply:
Fixed manually!